jcoin.datacommons.io/manifests/cronjobs/cronjobs.json

```diff
@@ -1,3 +1,3 @@
 {
-    "etl": "0 */6 * * *"
+    "etl": "0 */1 * * *"
```

Comment:
```suggestion
    "etl": "0 * * * *"
```

Reply:
maybe we could schedule it every hour during the day and skip it during the night 🤔 if this is a long-term thing and cost is an issue. for now it's probably fine though

Reply:
Ok thanks